src/widget/intrinsic_size.rs

```diff
@@ -0,0 +1,98 @@
+use embedded_graphics::prelude::*;
+
+#[derive(Clone, Copy, Eq, PartialEq)]
+#[cfg_attr(test, derive(Debug))]
+pub struct IntrinsicSize {
+    pub width: Option<u32>,
+    pub height: Option<u32>,
+}
```

Comment:
This represents the size a widget "wants" to be, absent other constraints. For example, it might not care at all (a container with no size and no children), or it might only care about one dimension (a container with no children, with a height set, but no width.)

Reply:
What's our fundamental unit, pixels?

Reply:
Yep (passing that through directly to the `embedded_graphics` code)